src/window/file_dialog.c

```diff
@@ -128,7 +128,17 @@ static struct {
 } data;
 
 static input_box main_input = {
-    16, 48, 18, 2, FONT_NORMAL_WHITE, 0, data.filter_text, FILTER_TEXT_SIZE, 0, input_box_changed
+    .x = 16,
+    .y = 48,
+    .width_blocks = 18,
+    .height_blocks = 2,
+    .font = FONT_NORMAL_WHITE,
+    .allow_punctuation = 0,
+    .text = data.filter_text,
+    .text_length = FILTER_TEXT_SIZE,
+    .put_clear_button_outside_box = 1,
+    .placeholder = 0,
+    .on_change = input_box_changed
```

Comment:
Since you're setting the properties individually, you don't need to set any to `0`. Static variables will always default to `0`.